ci/container_scripts/install_deps.sh

```diff
@@ -30,16 +31,26 @@ RPM_PACKAGES="
   make
   procps-devel
   python3
+  python3-pip
   xz
   xz-devel
   yum-utils
+  diffutils
   "
 
 case "$CONTAINER" in
     ubuntu:*|debian:*)
         sed -i '/deb-src/s/^# //' /etc/apt/sources.list
         DEBIAN_FRONTEND=noninteractive apt-get update
         DEBIAN_FRONTEND=noninteractive apt-get install -y $APT_PACKAGES
+        if [[ `python3 --version` == *" 3.5"* ]]; then
```

Comment:
Please leave a comment that this is for ordered dictionaries for `src/tools/convert.py`